app/src/main/java/org/mozilla/focus/browser/BrowserViewModel.kt

```diff
@@ -89,4 +93,8 @@ class BrowserViewModel(
     fun fullscreenChanged(isFullscreen: Boolean) {
         sessionRepo.fullscreenChange(isFullscreen)
     }
+
+    companion object {
+        @VisibleForTesting(otherwise = PRIVATE) val FULLSCREEN_BACKGROUND_DEFERRED_DISABLE_MILLIS: Long = 5000
```

Comment:
Optional nit: I feel like this should be at the top of the class, but I can't remember whether or not that's explicitly stated in kt style guides.

Reply:
I prefer to put them at the bottom: if companion objects ever start sprouting functions, I think it obscures the primary intent of the class, which is usually encapsulated in the instances.

I don't recall if it was referenced in the style guide.

Reply:
@Baron-Severin Official kotlin style guide actually says to put them at the end (but before inner classes): https://kotlinlang.org/docs/reference/coding-conventions.html#class-layout